plugin/web/extensions/sidebar/modals/datacatalog/components/content/YourDataPage/WebDataTab.tsx

```diff
@@ -57,34 +46,23 @@ const WebDataTab: React.FC<Props> = ({ onOpenDetails, setSelectedWebItem }) => {
   }, []);
 
   const handleClick = useCallback(async () => {
-    const result = await fetchDataFromUrl(dataUrl);
-    if (result) {
-      // Catalog Item
-      const filename = dataUrl.substring(dataUrl.lastIndexOf("/") + 1);
-      const id = "id" + Math.random().toString(16).slice(2);
-      const item: UserDataItem = {
-        type: "item",
-        id: id,
-        dataID: id,
-        description:
-          "Please contact the provider of this data for more information, including information about usage rights and constraints.",
-        name: filename,
-        url: dataUrl,
-        format: setDataFormat(fileType, filename),
-      };
-      const requireLayerName = needsLayerName(dataUrl);
-      if (onOpenDetails) onOpenDetails(item, requireLayerName);
-      if (setSelectedWebItem) setSelectedWebItem(item);
-    }
-  }, [
-    dataUrl,
-    fetchDataFromUrl,
-    fileType,
-    needsLayerName,
-    onOpenDetails,
-    setDataFormat,
-    setSelectedWebItem,
-  ]);
+    // Catalog Item
+    const filename = dataUrl.substring(dataUrl.lastIndexOf("/") + 1);
+    const id = "id" + Math.random().toString(16).slice(2);
```

Comment:
You can use the `generateID` function for id creation ;)

Reply:
Thanks, I'll make a separate PR for it.